app/controllers/spree/admin/orders_controller.rb

```diff
@@ -119,8 +119,13 @@ def order_params
 
       def load_order
         if params[:id]
-          @order = Order.includes(:adjustments, :shipments, line_items: :adjustments).
-            find_by!(number: params[:id])
+          begin
+            @order = Order.includes(:adjustments, :shipments, line_items: :adjustments).
+              find_by!(number: params[:id])
+          rescue ActiveRecord::RecordNotFound
+            @order = Order.includes(:adjustments, :shipments, line_items: :adjustments).
+              find_by!(id: params[:id])
```

Comment:
I'm wondering why this comes here? I mean, why we need to retrieve order via `id` when RecordNotFound via `number`. Does this relate in some way to this PR?